src/app/products.js

```diff
@@ -0,0 +1,40 @@
+
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+const fetchAllProducts = createAsyncThunk(
+  'products/fetchAllProducts',
+  async () => {
+    const requestResponse = await fetch(
+      'https://6x8prpit9f.execute-api.us-east-1.amazonaws.com/api/products',
+      {
+        method: 'GET',
+        headers: {
+          accept: 'application/json',
+          'x-api-key': '94tOOmgSAz3V6Xe7f2cMuz6qjgrIXS2xq8Xbly00',
```

Comment:
it's not a good practice to expose the API Key, try to use .env file and add it to gitignore file.